findutils/plan.sh

```diff
@@ -22,10 +22,34 @@ pkg_build_deps=(
   core/make
   core/gcc
   core/sed
+  core/pkg-config
+  core/m4
+  core/autoconf
+  core/automake
 )
 pkg_bin_dirs=(bin)
 
+do_prepare() {
+  # Glibc 2.28 removed headers that findutils expects.
+  # These patches correct findutils to allow it to build with
+  # the latest glibc.
+  # Thanks to Arch Linux for pulling these together
+  # https://git.archlinux.org/svntogit/packages.git/tree/trunk?h=packages/findutils
+  patch -p1 < "$PLAN_CONTEXT"/correct-glibc-fflush.patch
+  patch -p1 < "$PLAN_CONTEXT"/correct-glibc-makedev.patch
+  
+  # The Makefiles were generated with aclocal-1.14, and the above patches
+  # force it to want to regenerate. The following four lines can be removed
+  # if findutils releases a new version that no longer requries the patches
+  ACLOCAL_PATH="$ACLOCAL_PATH:$(pkg_path_for core/autoconf)/share/autoconf"
+  ACLOCAL_PATH="$ACLOCAL_PATH:$HAB_CACHE_SRC_PATH/$pkg_dirname/m4"
+  export ACLOCAL_PATH
+  aclocal
+}
+
 do_build() {
+
+  automake
```

Comment:
Question - how come `do_check` doesn't need the extra steps that `do_build` does?

Reply:
It executes `do_build` then `do_check`.